### DISCUSSION TITLE
Unified Audit Log Check

### DISCUSSION THREAD

**Discussion by cjh-nz** [maester365/maester#465] [Ideas]
We've noticed a few orgs with this disabled in recent security reviews. 

Might be worth including a check for it will look to get a custom detection written in the near future, buts it's a key baseline imho

**Comment by soulemike** [comment]
Hey @cjh-nz, would you think we'd want to check for more than the following:

https://github.com/maester365/maester/blob/main/powershell/public/cisa/exchange/Test-MtCisaAuditLog.ps1
https://github.com/maester365/maester/blob/main/powershell/public/cisa/exchange/Test-MtCisaAuditLogPremium.ps1
https://github.com/maester365/maester/blob/main/powershell/public/cisa/exchange/Test-MtCisaAuditLogRetention.ps1
